### ISSUE TITLE
Print a log in addition to the json results

### ISSUE THREAD

**Issue by gpetiot** [ocurrent/current-bench#321]
This is the follow-up of https://github.com/ocurrent/current-bench/issues/285, I would like to have a quite verbose log for the functions I'm benchmarking and unfortunately everything is parsed (at least attempted) as json by current-bench.

Here is my code: https://github.com/ocaml-ppx/ocamlformat/pull/2019
Here is the log I obtained: http://autumn.ocamllabs.io:8081/job/2022-02-21/205537-ocluster-build-ad032b

The JSON part at the beginning, and as you can see current-bench displays a parsing error when checking the benchmark results. Is there some kind of syntax to make the distinction clear, or is this not supported for now?

**Comment by art-w** [comment]
Thanks a lot for the report! We made some recent changes which should help for this situation and have redeployed... but please let us know if you still encounter an error!

We'll keep this issue open as it shows that our parsing logic is too simplistic: When we see a `{` at the beginning of a line, we grab everything that follows as a potential json and end up selecting this part of the log:
```ocaml
{
  s =
    (module struct
      type t = int

      let x = 1
    end);
}
```

**Comment by ElectreAAS** [comment]
I believe this should be solved by #365, @art-w can you confirm that?

**Comment by art-w** [comment]
Indeed!
